youtube_dl/extractor/enssavoirs.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class ENSSavoirsIE(InfoExtractor):
```

Comment:
jwplayer extraction code in generic extractor should be improved instead of adding this new extractor.